docs/source/conf.py

```diff
@@ -76,7 +76,7 @@
 # a list of builtin themes.
 #
 #html_theme = 'alabaster'
-html_theme = 'sphinxdoc'
+#html_theme = 'sphinxdoc'
```

Comment:
Rather than having commented out code, maybe have a list of theme's that have been used? And some notes on them as comments?

Reply:
I put a link to information on Sphinx themes which list all the themes.

Reply:
I see the comment about the list of themes. The problem is that it says "see the documentation for the list", in the context of this file -- I don't know what that means. Where do I find this list? I actually looked for that list and didn't see it anywhere. Maybe it's part of the documentation on readthedocs?

Part of what I'm getting at here is a personal nitpick -- I don't like to see commented out code especially when the code is in a repository. I think there are times when it is useful to do, but mostly it's not, you can use git to query what the previous code looked like. However, there are also times in a situation like this where it might be useful to have commented out code for a couple of the valid options. However, I'd add more comments to say why you might uncomment one of the options. For example

``` fortran
#html_theme = 'sphinxdoc'    # Uncomment and use this theme is a situation like this ______ 
```

I think that kind of thing can be useful. However, most of the time I'd say you shouldn't keep commented out code in a github repository. IMHO.

Reply:
yes, I agree with commented out codes, often distracting for me.  I further modified the comments.